lightningd/peer_control.c

```diff
@@ -1367,78 +1404,89 @@ static struct command_result *param_msat_u32(struct command *cmd,
 	return NULL;
 }
 
+void set_channel_fees(
```

Comment:
I think this should be static?  Plus, minor style variation: we don't usually end line  on '('

Reply:
thx, addressed 
96be8d6